src/pkg/bundle/deploy_test.go

```diff
@@ -429,3 +431,95 @@ func TestHelmOverrideVariablePrecedence(t *testing.T) {
 		})
 	}
 }
+
+func TestFileVariableHandlers(t *testing.T) {
```

Comment:
Love the unit tests, what do you think of adding to `TestHelmOverrideVariablePrecedence` to test the full code path starting in `processOverrideVariables`?